py3dtiles/gltf.py

```diff
@@ -55,19 +55,21 @@ def from_array(array):
         if struct.unpack("4s", array[0:4])[0] != b"glTF":
             raise RuntimeError("Array does not contain a binary glTF")
 
-        if struct.unpack("i", array[4:8])[0] != 1:
+        version = struct.unpack("i", array[4:8])[0]
+        if version != 1 and version != 2:
```

Comment:
Is this an "and test" or an "or test" ? (funnily hard to read sentence, to me at least)

Reply:
My bad. Forget that.